### DISCUSSION TITLE
Where can I go to ask for help with vermin?

### DISCUSSION THREAD

**Discussion by hj-ross** [netromdk/vermin#82] [Q&A]
I'm attempting to use vermin as part of a migration of a large number of scripts from python2 to python3 (long overdue!). I'm using a Windows system, and have attempted to use vermin from a variety of environments (cmd, powershell, Spyder, Jupyter), and I continue to encounter the following error:

> python
Python 3.9.7 (default, Sep 16 2021, 16:59:28) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import vermin
>>> vermin test.py
  File "<stdin>", line 1
    vermin test.py
               ^
SyntaxError: invalid syntax

Could you advise me on where to find assistance on these kinds of problems? I can't tell if I'm calling it incorrectly or if it's something else.

**Comment by netromdk** [comment]
Hello @hj-ross. The normal way to use Vermin is as a command-line program. If you've installed Vermin already, you can run your command without first starting Python, as you've done in your example.
```
vermin test.py
```

**Reply by netromdk** [reply]
Did you solve your issue?
